content/user/first_steps/server_requirements.md

```diff
@@ -64,7 +64,7 @@ Zen Cart requires a few PHP modules/extensions installed:
  - `gd`
  - `getimagesize()`
  - `iconv`
- - `intl`  (required since v2.0.0)
```

Comment:
I'd suggest leaving this as 2.0.0 since that's when it started being officially required.